module-2/backend/week_one.md

```diff
@@ -1,32 +1,159 @@
 ## Week One - Module 2 Recap
 
-Fork this respository. Answer the questions to the best of your ability. Try to answer them with limited amount of external research. These questions cover the majority of what we've learned this week (which is a TON!). 
-
-Note: When you're done, submit a PR. 
-
 1. List the five common HTTP verbs and what the purpose is of each verb.
+  <br>
+  GET-Retrieves information
+  <br>
+  POST-Send information/manipulates data

```

Comment:
POST requests are intended to create new records in the database and PUT requests are intended to update a specific record in the database